docs/static/ds6/docs.min.js

```diff
@@ -120,7 +120,7 @@ $_mod.def("/makeup-screenreader-trap$0.1.4/index",function(e,t,n,r,a){"use stric
"dialog"};e.exports=function(){function t(n,e){l(this,t),this._options=o({},C,e);var i=this._options.dialogBaseClass;this._el=n,this._isModal="true"===this._el.getAttribute("aria-modal"),this._hasTransitions=!!this._el.dataset&&"true"===this._el.dataset.makeupDialogHasTransitions,this._windowEl=this._el.querySelector(".".concat(i,"__window")),this._closeButtonEl=this._el.querySelector(".".concat(i,"__close")),this._confirmButtonEl=this._el.querySelector(".".concat(i,"__confirm")),this._rejectButtonEl=this._el.querySelector(".".concat(i,"__reject")),this._onCloseButtonClickListener=v.bind(this),this._onConfirmButtonClickListener=d.bind(this),this._onRejectButtonClickListener=_.bind(this),this._onKeyDownListener=f.bind(this),this._onOpenTransitionEndListener=h.bind(this),this._onCloseTransitionEndListener=u.bind(this),this._el.classList.add("".concat(i,"--js")),this._isModal&&this.open&&(document.body.classList.add("has-modal"),r(this),E.modal(this._el)),this.observeEvents()}return c(t,[{key:"unobserveEvents",value:function(){this._el.removeEventListener("click",this._onCloseButtonClickListener),document.removeEventListener("keydown",this._onKeyDownListener),this._confirmButtonEl&&this._confirmButtonEl.removeEventListener("click",this._onConfirmButtonClickListener),this._rejectButtonEl&&this._rejectButtonEl.removeEventListener("click",this._onRejectButtonClickListener)}},{key:"observeEvents",value:function(){!0!==this._destroyed&&(this._closeButtonEl.addEventListener("click",this._onCloseButtonClickListener),document.addEventListener("keydown",this._onKeyDownListener),this._confirmButtonEl&&this._confirmButtonEl.addEventListener("click",this._onConfirmButtonClickListener),this._rejectButtonEl&&this._rejectButtonEl.addEventListener("click",this._onRejectButtonClickListener))}},{key:"destroy",value:function(){this._destroyed=!0,this.unobserveEvents(),this._onCloseButtonClickListener=null,this._onConfirmButtonClickListener=null,this._onRejectButtonClickListener=null,this._onKeyDownListener=null,this._onOpenTransitionEndListener=null,this._onCloseTransitionEndListener=null}},{key:"focusables",get:function(){return m(this._windowEl)}},{key:"open",get:function(){return!this._el.hidden},set:function(t){!0===t&&!0!==this.open?this._hasTransitions?(this.unobserveEvents(),this._cancelTransition&&this._cancelTransition(),this._cancelTransition=p(this._el,"".concat(this._options.dialogBaseClass,"--show"),this._onOpenTransitionEndListener)):(this._el.hidden=!1,this._isModal&&(r(this),E.modal(this._el)),this._el.dispatchEvent(new CustomEvent("".concat(this._options.dialogBaseClass,"-open")))):!1===t&&!1!==this.open&&(this._hasTransitions?(this.unobserveEvents(),this._cancelTransition&&this._cancelTransition(),this._cancelTransition=p(this._el,"".concat(this._options.dialogBaseClass,"--hide"),this._onCloseTransitionEndListener)):(E.unmodal(),this._el.hidden=!0,this._el.dispatchEvent(new CustomEvent("".concat(this._options.dialogBaseClass,"-close")))))}}]),t}()});
```

Comment:
Why did this change. I dont think it should

Reply:
I updated the `dialog.js` class.